### ISSUE TITLE
imgadult, imgdrive, imgwallet, imgtaxi - does not work

### ISSUE THREAD

**Issue by elvisef** [Owyn/HandyImage#388]
Hey.

Just checked these two, and these do not seem to work properly.

ex link: [https://imgadult.com/img-5eb394f518931.html](https://imgadult.com/img-5eb394f518931.html)

Can U confirm and eventually fix it, please ?
Tried to disabled all of addons You see on my screenshot, but didn't work.

![firefox_ew71rFj7bV](https://user-images.githubusercontent.com/25976749/103142198-a132d500-46ff-11eb-959c-3868c7190564.png)

Just to be more precise, it freezes before changing the "Loading... Please wait." to "Click.. blabla", but it never changes while the HandyImages is loaded. NoScript accepts everything on these pages. Ubloc, Html5 disabler and Ghostery is turned off - no change.

**Comment by Owyn** [comment]
Works fine for me
![изображение](https://user-images.githubusercontent.com/1309656/103142493-00e7ab00-4715-11eb-95f8-32ec1fb48e78.png)

Check that javascript is turned on for the page

**Comment by elvisef** [comment]
Alright, I've just turned off everything - every little addon in my browser (FF). No results.
But a lot of console errors.

It turns out that "closeOverlay()" function (it's in imgtaxi) has something with tracking, that does not load at all.
Moreover there is a jQuery issue (?)

Please look into it, I am wondering why this issue happens.

![2021-01-04_08-25-02](https://user-images.githubusercontent.com/25976749/103511116-b2e64c00-4e66-11eb-8fbf-c808164b61ab.png)
![explorer_2021-01-04_08-22-27](https://user-images.githubusercontent.com/25976749/103511118-b37ee280-4e66-11eb-945a-518e3c8ba49b.png)
![2021-01-04_08-20-35](https://user-images.githubusercontent.com/25976749/103511121-b37ee280-4e66-11eb-8efd-5522eaf73212.png)
![2021-01-04_08-18-21](https://user-images.githubusercontent.com/25976749/103511137-baa5f080-4e66-11eb-9768-d7c03b49e7de.png)

**Comment by Owyn** [comment]
`$ is not defined` means whole jquery on the website could not load at all, and overlay is closed via jquery it seems

doesn't show any errors for me : 
![изображение](https://user-images.githubusercontent.com/1309656/103552822-90335200-4ebd-11eb-96be-1f69a67afdc1.png)

instead of disabling everything (advanced tracking protection is still on it shows btw (left to address bar icon), just try it from a clean firefox profile - it's fast and safe (won't affect your current profile): https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles#w_starting-the-profile-manager 

plus, make sure you have checked that the website itself works for you **without** handyimage

**Comment by elvisef** [comment]
This gets really more complicated, yeah : ) But I need to know if this is a bug on my side only, or not.
Alright, going further..

Let's base on the first link a pasted on my first post: [https://imgadult.com/img-5eb394f518931.html](url) (because Ive tried imgtaxi as welll but just to not confuse it all).

I tried a new profile option in FF - thanks, with this I can also test on fresh new profile without closing the old/active one - nice !

So firstly, I tried what happens without any addons installed and without a HandyImage script loaded (but greasemonkey is installed and HI is disabled). These are screenshots:
![2021-01-05_09-43](https://user-images.githubusercontent.com/25976749/103625151-dbd81100-4f3a-11eb-832b-bb898120b75c.png)
![2021-01-05_09-43_1](https://user-images.githubusercontent.com/25976749/103625205-f1e5d180-4f3a-11eb-8b48-6b8de455c540.png)
So as I see, the image is being loaded successfully - yes, with shitty adds, and so on... massacre.

But... with just HandyImage loaded, nothing else, it looks still the same. Look:
![2021-01-05_09-39](https://user-images.githubusercontent.com/25976749/103625363-2ce80500-4f3b-11eb-82df-782244c16616.png)
![2021-01-05_09-49](https://user-images.githubusercontent.com/25976749/103625441-45581f80-4f3b-11eb-8dd1-8a9010429bba.png)

These are two screenshots, because I wanted You to see what errors exactly are being showed.
_qaq is probably something with tracking from.. Google ? or whatever.. but it is in the function I wrote earlier about.

JQuery - hmm.... it's really weird... that it's not being loaded.

But wondering what about the line that says: "Uncaught Error: Permission denied to access object" - something is still blocked ?
What else can be blocking it...

FF protection is off, I disabled it.

**Comment by Owyn** [comment]
When it works for you - you still get some errors about ads being blocked `ads.exoclick.com`, is it not something still blocking ads for you (firewall, dns)? 

someone referenced this issue in their https://github.com/bogachenko/fuckfuckadblock/commit/f1e671dcb9f95781b07d8ea92a3ea97b28a0ca03 claiming it's now fixed (so it was caused by their adblocker?)

![изображение](https://user-images.githubusercontent.com/1309656/103647680-bebc3600-4f6c-11eb-9b24-d0a46ead26ae.png)

**Comment by elvisef** [comment]
> someone referenced this issue in their [bogachenko/fuckfuckadblock@f1e671d](https://github.com/bogachenko/fuckfuckadblock/commit/f1e671dcb9f95781b07d8ea92a3ea97b28a0ca03) claiming it's now fixed (so it was caused by their adblocker?)

I tried it, but didn't work at all. A do not use AdBlock, I use Qblock. But after subscribing, nothing changed.

There should be a way to enter the image, WITHOUT allowing ANY ads or tracking services to load.

Anyway, I tried TamperMonkey addon (just removed GreaseMonkey) and geuss what ?: )
Yes... it worked like a charm ! I do not know why, how... it works. For now it works.

Testing on the "imgadult" link I wrote above. And I can see a big image version.
But... 

On the first try - to rowks, it loads - screenshot:
![explorer_2021-01-06_16-25-39](https://user-images.githubusercontent.com/25976749/103786114-58064d80-503c-11eb-9a65-5c147389661a.png)

But the second try - when I enter the URL by clicking in the address bar, or using CTRL + R or F5 keys, is hmm... look:
![firefox_2021-01-06_16-26-15](https://user-images.githubusercontent.com/25976749/103786141-618fb580-503c-11eb-86d0-6b254ec6ef4d.png)

So it doesn't work if I try to reload it, but... not always. Sometimes it works after reloading.
An whats more, after reload, if I need to click the "magic" button to show the image, it shows me the image itself, so the whole ads stuff is gone, but this button... I need to click it though.

Wondering why it sometimes reloads well and sometimes not.
Maybe my screenshots can tell You someting.
I've tested it with disabled QBlock and Ghostery, but after enabling them, it is working same, showing the image.
Only this reloading is a bit confusing.
